server/core/config/raw/temporal.go

```diff
@@ -7,21 +7,22 @@ import (
 )
 
 type Temporal struct {
-	Port string `yaml:"port" json:"port"`
-	Host string `yaml:"host" json:"host"`
+	Port            string `yaml:"port" json:"port"`
+	Host            string `yaml:"host" json:"host"`
+	UseSystemCACert bool   `yaml:"us_system_ca_cert" json:"us_system_ca_cert"`
 }
 
 func (t *Temporal) Validate() error {
 	return validation.ValidateStruct(t,
 		validation.Field(&t.Host, validation.Required),
 		validation.Field(&t.Port, validation.Required),
-		validation.Field(&t.Host, is.IP),
```

Comment:
We should be able to pass dns records as well since that's how our cluster endpoint is defined.